TestAssets/TestProjects/DotNetCoreTestProject/project.json

```diff
@@ -0,0 +1,21 @@
+{
+  "dependencies": {
+    "MSTest.TestAdapter": "1.0.3-preview",
```

Comment:
How are these packages updated between builds? Does the patch number increase every time?